app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_senter: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_senter = distance_from_city_senter
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> None:
+        total_price = 0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_price += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_price, 1)
+
```

Comment:
No need to round the total_price value, you already did that in the calculate_washing_price method.